build/miengine.targets

```diff
@@ -47,6 +47,22 @@
     </ItemGroup>
   </Target>
 
+  <!-- Custom target that will be invoked after signing and loc to copy signed satellite assemblies
```

Comment:
I think this would be better either in its own targets file or in DropFiles.targets. This file doesn't have the MicroBuild props that you depend on.

Reply:
Sounds reasonable.